src/schema/v2/sale/index.ts

```diff
@@ -257,16 +257,24 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
       },
       saleArtworksConnection: {
         type: saleArtworkConnection,
-        args: pageable(),
+        args: pageable({
+          internalIDs: {
+            type: new GraphQLList(GraphQLString),
+            description: "List of sale artwork internal IDs to fetch",
+          },
+        }),
         resolve: (sale, options, { saleArtworksLoader }) => {
           const { limit: size, offset } = getPagingParameters(options)
+          const { internalIDs: ids } = options
+
           return saleArtworksLoader(sale.id, {
             size,
             offset,
+            ids,
           }).then(({ body }) =>
             connectionFromArraySlice(body, options, {
               arrayLength: sale.eligible_sale_artworks_count,
```

Comment:
I think with this param, if you use `ids` the connection might think you have a next page, when you don't.

Since this is unlikely to be _actually_ paginated, I would make it `{sliceStart: 0, arrayLength: body && body.length}` (you've already done the `sliceStart` one), in the case of an `ids` param.